components/gcp/ml_engine/train/README.md

```diff
@@ -0,0 +1,135 @@
+
+# CloudML - Train
+
+Submits a Cloud Machine Learning training job.
```

Comment:
Recommend replacing with "A Kubeflow Pipeline component to submit a Cloud Machine Learning Engine training job as a step in a pipeline"

Reply:
Done